netty/src/main/java/io/grpc/netty/NettyServerStream.java

```diff
@@ -53,6 +58,13 @@
   private final NettyServerHandler handler;
   private final Http2Stream http2Stream;
   private final WriteQueue writeQueue;
+  private final Attributes attributes;
+
+  private static Attributes.Key<SocketAddress> REMOTE_ADDR_KEY =
+      Attributes.Key.named("remote-addr");
```

Comment:
Adding is not an API-breaking change. I think that would be fine. A transport-neutral location is preferred to a netty-specific location.